ElrondNetwork/api.elrond.com
# Add batch transactions endpoints

## Description of the reasoning behind the pull request (what feature was missing / how the problem was manifesting itself / what was the motive behind the refactoring)
- Transactions could't be processed in batches depending on address
  
## Proposed Changes
- Add a cronjob that that watch transactions batches and update batch status
- Add endpoint in order to start a batch
- Add endpoint in order to show batch status

## How to test
- Create a JWT with longer expiry time
- Make a few swaps on dex
- Check their status using local instance of api
